ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -3185,7 +3185,7 @@ public void loadTable(Path loadPath, String tableName, LoadFileType loadFileType
     //column stats will be inaccurate
     if (resetStatistics) {
       LOG.debug("Clearing table statistics for " + tbl.getDbName() + "." + tbl.getTableName());
-      StatsSetupConst.clearColumnStatsState(tbl.getParameters());
+      StatsSetupConst.setBasicStatsState(tbl.getParameters(),StatsSetupConst.FALSE);
```

Comment:
Can be combined with previous if block which also does the same thing. Also, the comment says "column stats will be inaccurate" but we are resetting both basic and column stats. Correct the log message and comment accordingly.